File: 백준/Bronze/6502. 동혁 피자/동혁 피자.cc
#include <iostream>
#include <cmath>
using namespace std;
int main(void) {
    int r,w,l,cnt=1;
    while(1) {
        cin>>r;
        if(r==0) break;
        cin>>w>>l;
        cout<<"Pizza "<<cnt++<<(2*r>=sqrt(w*w+l*l)?" fits ":" does not fit ")<<"on the table.\n";
    }
    return 0;
}

File: 백준/Bronze/6502. 동혁 피자/동혁 피자.java
import java.util.Scanner;
import java.lang.Math;
public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int cnt=1;
        while(true) {
            int r = scan.nextInt();
            if(r==0) break;
            int w = scan.nextInt();
            int l = scan.nextInt();
            System.out.printf("Pizza %d %s on the table.\n",cnt++,2*r>=Math.sqrt(w*w+l*l)?"fits":"does not fit");
        }
    }
}